CogNative/main.py

```diff
@@ -32,8 +32,25 @@
     print(colorize("ERROR: Enter an output .wav file", "error"))
     exit(1)
 
+v.set_file_path(file_path)
+
+embedding = input("Use embedding? (y/n)\n")
+if embedding.lower() in ['y', 'yes']:
+    embedding_path = v.get_embedding_path()
+    if not Path(embedding_path).exists():
+        print(colorize(f"ERROR: Embedding file ({embedding_path}) not found.", "error"))
+        exit(1)
+    if not Path(embedding_path).suffix == '.ckpt':
+        print(colorize("ERROR: Enter a .ckpt embedding file", "error"))
+        exit(1)
+else:
+    embedding_path = None
+
 # ENCODE (sometimes this takes time, so it is after inputs)
-v.encode_voice(file_path)
+if embedding_path:
+    v.load_embedding(embedding_path)
+else:
+    v.encode_voice(file_path, save_embedding=True)
```

Comment:
```save_embedding=True``` : This means that the default will be to save the embedding. 

I could add another input option, but I don't know that it is necessary. All of these inputs will likely be changed to sys arguments soon and then we could make it: ```python main.py --input <audio.wav> --output <output.wav> --save-embedding --text <text/text_file>```